### ISSUE TITLE
Add a config file to enable/disable various features

### ISSUE THREAD

**Issue by not-an-aardvark** [pokemontrades/flairhq#511]
This should make it easier to toggle optional features, and to change specific rules (e.g. whether or not to archive modmails, whether to count a specific trade type for flair, etc.), or  without having to change a bunch of code. The config file should be on gitignore, with a second "default" config file included in the repo.

This will require cleaning up the code to make features easily toggleable, which is probably a good idea anyway.

**Comment by not-an-aardvark** [comment]
[Draft of what the config file would look like](https://gist.github.com/not-an-aardvark/c970d6c60f81726a8535)

**Comment by YaManicKill** [comment]
What's wrong with using the existing config file?

**Comment by not-an-aardvark** [comment]
local.js? That also works.

I suppose it would be good if there was default config file that was part of the repo, and specific options could optionally be overridden in local.js.

**Comment by YaManicKill** [comment]
There actually is. The local.example.js is actually included

**Comment by not-an-aardvark** [comment]
Right, but I mean a separate default that actually gets used. So we would have a default config in a separate file, as:

```javascript
module.exports = {
  option1: true,
  option2: true,
  option3: true
};
```

Then in local.js, one could override the default parameters like this:
```javascript
module.exports = {
  /* ... (password stuff, etc.) ... */
  config: {
    option2: false // Override the default value of option2, but use the default values for option1 and option3
  }
};
```

**Comment by YaManicKill** [comment]
The local.example.js actually does get used.

**Comment by not-an-aardvark** [comment]
Oh, so it does. Never mind, then.
